src/main/java/core/basesyntax/strategy/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationHandler implements OperationHandler {
+
+    @Override
+    public void accept(String data) {
+        int indexOfSeparator = data.indexOf(OperationHandler.COMMA_SEPARATOR);
+        int amount = Storage.STORAGE.get(data.substring(0, indexOfSeparator));
+        Storage.STORAGE.put(data.substring(0, indexOfSeparator),
+                amount + Integer.parseInt(data.substring(indexOfSeparator + 1)));
```

Comment:
i think you need to separate this method and 3 other, because you have here two purpose: parsing and pushing data into storage. so it's better to create class which will parse data to some fruit object with some fields(operation, fruitName, quantity)